R/graded.R

```diff
@@ -360,7 +362,10 @@ pass_if_equal <- function(
     assert_object_found_in_env(y, env, "pass_if_equal")
   }
   maybe_extras(
-    grade_if_equal(x = x, y = y, message = message, correct = TRUE, env = env, ...),
+    grade_if_equal(
+      x = x, y = y,
+      message = message, correct = TRUE, env = env, tolerance = tolerance, ...
+    ),
```

Comment:
Style nit: arguments all on the same line or one line each

```suggestion
    grade_if_equal(
      x = x, 
      y = y,
      message = message,
      correct = TRUE,
      env = env,
      tolerance = tolerance, 
      ...
    ),
```

```suggestion
    grade_if_equal(x = x, y = y, message = message, correct = TRUE, env = env, tolerance = tolerance, ...),
```

Reply:
(same for  `fail_if_equal()` and `grade_if_equal()`)

Reply:
Updated in [cc12cf8](https://github.com/rstudio/gradethis/pull/295/commits/cc12cf8653c78518e9444a68ee07f4e31db7eefd)